### ISSUE TITLE
feat: add react-native support

### ISSUE THREAD

**Issue by levi-pires** [amzn/style-dictionary#497]
I would like to know if you would accept a PR to add support for React Native.

These are the changes I made. It´s all commited in my [fork](https://github.com/Meiuca/style-dictionary)

```js
// lib/common/transformGroups.js: line 283
'react-native': [
    'name/cti/camel',
    'size/object',
    'color/css'
  ]
```

```js
// lib/common/transforms.js: line 543
"size/object": {
    type: "value",
    matcher: isSize,
    transformer: function (prop) {
      var val = parseFloat(prop.value);
      if (isNaN(val)) throwSizeError(prop.name, prop.value, "object");

      return {
        original: prop.value,
        number: val,
        decimal: val / 100,
        scale: val * 16,
      };
    },
  }
```

**Comment by dbanksdesign** [comment]
I think this looks good! Do you think you would be able to provide an example project as well in the examples directory? You can take a look at #320 to see how we added Flutter support. One question about the 'size/object' transform, how is that used in a React Native context?

**Comment by levi-pires** [comment]
> 
> 
> I think this looks good! Do you think you would be able to provide an example project as well in the examples directory? You can take a look at #320 to see how we added Flutter support. One question about the 'size/object' transform, how is that used in a React Native context?
> 
> If you are keen to open a PR, please do it on the 3.0 branch so it will go out in the next release. Thanks!

The `scale` can be used when the original value is a rem value, the `decimal` can be used if the original value is a percentage value but React Native wants a float, the `number` can be used when it is a simple value but React Native wants a number and `original` can be used when the other scenarios don't fit.

**Comment by chazzmoney** [comment]
This is now merged and will be included in the 3.0 release of Style Dictionary. Thanks Levi!
